roxctl/netpol/generate/command.go

```diff
@@ -0,0 +1,53 @@
+package generate
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/printer"
+)
+
+type netpolGenerateCommand struct {
+	// Properties that are bound to cobra flags.
+	offline    bool
+	folderPath string
+
+	// injected or constructed values
+	env     environment.Environment
+	printer printer.ObjectPrinter
+}
+
+// Command defines the central command tree
+func Command(cliEnvironment environment.Environment) *cobra.Command {
+	netpolGenerateCmd := &netpolGenerateCommand{env: cliEnvironment}
+	c := &cobra.Command{
+		Use: "generate <folder-path>",
+		RunE: func(c *cobra.Command, args []string) error {
+			if err := validate(args); err != nil {
+				return err
+			}
+			if err := netpolGenerateCmd.construct(args); err != nil {
+				return err
+			}
+			return netpolGenerateCmd.generateNetpol()
+		},
+	}
+
+	c.Flags().BoolVarP(&netpolGenerateCmd.offline, "offline", "", false, "whether to connect to a central instace for additional information")
```

Comment:
Why `BoolVarP` if you're not adding a shorthand? Either add a `o` shorthand or use `BoolVar`

Reply:
I originally thought we might add a shorthand, but we can switch that over if we need it. Done!